addons/binding/org.openhab.binding.yamahareceiver2/src/main/java/org/openhab/binding/yamahareceiver2/discovery/ZoneDiscoveryService.java

```diff
@@ -0,0 +1,61 @@
+package org.openhab.binding.yamahareceiver2.discovery;
```

Comment:
Could you run a `mvn license:format` to make sure you have a license header on all the files?